tq/transfer_queue.go

```diff
@@ -633,7 +634,12 @@ func (q *TransferQueue) run() {
 
 // canRetry returns whether or not the given error "err" is retriable.
 func (q *TransferQueue) canRetry(err error) bool {
-	return errors.IsRetriableError(err)
+	switch cause := errors.Cause(err).(type) {
```

Comment:
@technoweenie Initially I had it in `errors.IsRetriableError` but then I moved it to `canRetry`, because it seems that `errors.IsRetriableError`, together with the other functions `types.go`, are about type inference, and not about flow control. I don't know the codebase and the conventions there well so I am ok to move it there if you are sure this is the place.